packages/video_player/video_player_avfoundation/test/avfoundation_video_player_test.dart

```diff
@@ -135,6 +135,18 @@ void main() {
       expect(textureId, 3);
     });
 
+    test('create with incorrect asset', () async {
```

Comment:
I'd rename this to something like 'create with incorrect asset throws exception' or something like that.